PowerNSX.psm1

```diff
@@ -5104,11 +5104,12 @@ function New-NsxController {
     The New-NsxController cmdlet deploys a new NSX Controller.
     
     .EXAMPLE
+    $ControllerName = "MyNSXCtrl1"
```

Comment:
Would prefer a separate example using controller name - just because its not mandatory - dont want to imply to the casual user that it is.  Just add a second .Example section.